services/navigation/builtin/builtin.go

```diff
@@ -391,7 +391,6 @@ func (svc *builtIn) SetMode(ctx context.Context, mode navigation.Mode, extra map
 			return errors.New("explore mode requires at least one vision service")
 		}
 		svc.startExploreMode(cancelCtx)
-		return errors.New("navigation mode 'explore' is not currently available")
```

Comment:
[note] Do we want to enable explore mode in this PR or after testing. Note: If it is after testing we will need to disable `TestExplore`